Gemfile.lock

```diff
@@ -104,55 +104,55 @@ GEM
       mini_mime (>= 0.1.1)
     marcel (0.3.3)
       mimemagic (~> 0.3.2)
-    method_source (0.9.2)
+    method_source (1.0.0)
     mimemagic (0.3.4)
     mini_mime (1.0.2)
     mini_portile2 (2.4.0)
     minitest (5.14.0)
     msgpack (1.3.3)
     nio4r (2.5.2)
-    nokogiri (1.10.8)
+    nokogiri (1.10.9)
       mini_portile2 (~> 2.4.0)
     popper_js (1.16.0)
     public_suffix (4.0.3)
-    puma (4.3.1)
+    puma (4.3.3)
       nio4r (~> 2.0)
     rack (2.2.2)
     rack-test (1.1.0)
       rack (>= 1.0, < 3)
-    rails (6.0.2.1)
-      actioncable (= 6.0.2.1)
-      actionmailbox (= 6.0.2.1)
-      actionmailer (= 6.0.2.1)
-      actionpack (= 6.0.2.1)
-      actiontext (= 6.0.2.1)
-      actionview (= 6.0.2.1)
-      activejob (= 6.0.2.1)
-      activemodel (= 6.0.2.1)
-      activerecord (= 6.0.2.1)
-      activestorage (= 6.0.2.1)
-      activesupport (= 6.0.2.1)
+    rails (6.0.2.2)
+      actioncable (= 6.0.2.2)
+      actionmailbox (= 6.0.2.2)
+      actionmailer (= 6.0.2.2)
+      actionpack (= 6.0.2.2)
+      actiontext (= 6.0.2.2)
+      actionview (= 6.0.2.2)
+      activejob (= 6.0.2.2)
+      activemodel (= 6.0.2.2)
+      activerecord (= 6.0.2.2)
+      activestorage (= 6.0.2.2)
+      activesupport (= 6.0.2.2)
       bundler (>= 1.3.0)
-      railties (= 6.0.2.1)
+      railties (= 6.0.2.2)
       sprockets-rails (>= 2.0.0)
     rails-dom-testing (2.0.3)
       activesupport (>= 4.2.0)
       nokogiri (>= 1.6)
     rails-html-sanitizer (1.3.0)
       loofah (~> 2.3)
-    railties (6.0.2.1)
-      actionpack (= 6.0.2.1)
-      activesupport (= 6.0.2.1)
+    railties (6.0.2.2)
+      actionpack (= 6.0.2.2)
+      activesupport (= 6.0.2.2)
       method_source
       rake (>= 0.8.7)
       thor (>= 0.20.3, < 2.0)
     rake (13.0.1)
     rb-fsevent (0.10.3)
     rb-inotify (0.10.1)
       ffi (~> 1.0)
-    regexp_parser (1.6.0)
+    regexp_parser (1.7.0)
```

Comment:
### [regexp_parser](https://github.com/ammar/regexp_parser)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.6.0 → 1.7.0

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **5** commits. See the full changes on [the compare page](https://github.com/ammar/regexp_parser/compare/v1.6.0...v1.7.0).

These are the individual commits:
- (05503f8) [Fix warnings in specs on Ruby 2.7](https://github.com/ammar/regexp_parser/commit/05503f861e5843cd46ccced978af14965c3254d9)
- (23ef248) [Stop running tests for very old Rubies](https://github.com/ammar/regexp_parser/commit/23ef248833914558c847f1d0696fc973484a2ed2)
- (74de202) [Make traverse/each_expression to return an Enumerator if the…](https://github.com/ammar/regexp_parser/commit/74de20210640ce8c56d82f63a25325c07d88b26d)
- (c4ee3a0) [Merge pull request #62 from pocke/return-enumerator](https://github.com/ammar/regexp_parser/commit/c4ee3a03bc8d59499675328a4db42784d0a04a7f)
- (b2adbaa) [Release v1.7.0](https://github.com/ammar/regexp_parser/commit/b2adbaafa0208656c91c50cf4667265ad222f357)


</details>